### ISSUE TITLE
install error arch - i386 with R 4.0.0  

### ISSUE THREAD

**Issue by mrhelmus** [r-lib/backports#31]
Sorry to bother. I am running R 4.0.0 with no other packages installed. I tried to install `devtools` but I received the error below. *Any idea what the below error means?* 

> install.packages("backports")

installing the source package ‘backports’

trying URL 'https://cloud.r-project.org/src/contrib/backports_1.1.7.tar.gz'
Content type 'application/x-gzip' length 17676 bytes (17 KB)
downloaded 17 KB

* installing *source* package 'backports' ...
** package 'backports' successfully unpacked and MD5 sums checked
** using staged installation
** libs

*** arch - i386
"c:/rtools40/mingw32/bin/"gcc  -I"C:/PROGRA~1/R/R-40~1.0/include" -DNDEBUG          -O2 -Wall  -std=gnu99 -mfpmath=sse -msse2 -mstackrealign -c dotsElt.c -o dotsElt.o
sh: c:/rtools40/mingw32/bin/gcc: No such file or directory
make: *** [C:/PROGRA~1/R/R-40~1.0/etc/i386/Makeconf:222: dotsElt.o] Error 127
ERROR: compilation failed for package 'backports'
* removing 'C:/Users/X/Documents/R/win-library/4.0/backports'

The downloaded source packages are in
        ‘C:\Users\X\AppData\Local\Temp\RtmpcHvAt5\downloaded_packages’
Warning message:
In install.packages("backports") :
  installation of package ‘backports’ had non-zero exit status

**Comment by Christoph999** [comment]
Perhaps, this helps: When installing without source, the error vanishes and installation seems to be successful:

```
> install.packages("backports", lib="C:/R/R-4.0.0/library")

  There is a binary version available but the source version is later:
          binary source needs_compilation
backports  1.1.6  1.1.7              TRUE

installing the source package ‘backports’

trying URL 'https://cran.rstudio.com/src/contrib/backports_1.1.7.tar.gz'
Content type 'application/x-gzip' length 17676 bytes (17 KB)
downloaded 17 KB

* installing *source* package 'backports' ...
** Paket 'backports' erfolgreich entpackt und MD5 Summen überprüft
** using staged installation
** libs

*** arch - i386
"c:/rtools40/mingw32/bin/"gcc  -I"C:/R/R-40~1.0/include" -DNDEBUG          -O2 -Wall  -std=gnu99 -mfpmath=sse -msse2 -mstackrealign -c dotsElt.c -o dotsElt.o
sh: c:/rtools40/mingw32/bin/gcc: No such file or directory
make: *** [C:/R/R-40~1.0/etc/i386/Makeconf:222: dotsElt.o] Error 127
ERROR: compilation failed for package 'backports'
* removing 'C:/R/R-4.0.0/library/backports'
Warning in install.packages :
  installation of package ‘backports’ had non-zero exit status

The downloaded source packages are in
	‘C:\Users\sqc\AppData\Local\Temp\Rtmp0wMNAd\downloaded_packages’
> install.packages("backports", lib="C:/R/R-4.0.0/library")

  There is a binary version available but the source version is later:
          binary source needs_compilation
backports  1.1.6  1.1.7              TRUE

trying URL 'https://cran.rstudio.com/bin/windows/contrib/4.0/backports_1.1.6.zip'
Content type 'application/zip' length 77609 bytes (75 KB)
downloaded 75 KB

package ‘backports’ successfully unpacked and MD5 sums checked

The downloaded binary packages are in
	C:\Users\sqc\AppData\Local\Temp\Rtmp0wMNAd\downloaded_packages
```
But devtools still throws an error:

```
devtools::test()
 Show Traceback
 
 Rerun with Debug
 Error: .onLoad in loadNamespace() für 'pkgload' fehlgeschlagen, Details:
  Aufruf: NULL
  Fehler: package ‘backports’ was installed before R 4.0.0: please re-install it 
```

**Comment by mrhelmus** [comment]
Thank you!! That worked for install, and I will post here if I find things working or failing once I build the next package.

**Comment by mllg** [comment]
It looks like your system is not properly set up to compile code for the 32bit arch. You could try to re-install or update rtools. Or just wait a couple of days for CRAN serving windows binaries.

**Comment by Christoph999** [comment]
The problem is solved in my case. I needed to update rtools40 (which I forgot). I also updated RStudio - it seems R 4.0 + old RStudio faces problems.
The only remaining problem (I think that is how my question above appeared can be found on SO:
https://stackoverflow.com/questions/61868910/does-rstudios-build-button-not-work-properly
At the beginning backport was an issue, but it turns out, that this is not the root cause...
